examples/lock-app/lock-common/lock-app.zap

```diff
@@ -1831,6 +1831,21 @@
               "maxInterval": 65534,
               "reportableChange": 0
             },
+            {
+              "name": "SerialNumber",
+              "code": 15,
+              "mfgCode": null,
+              "side": "server",
+              "included": 1,
+              "storageOption": "RAM",
```

Comment:
Why do we store a serial number in RAM? would a serial number not be fixed per device somewhere in flash?

Reply:
You're right, I mistakenly added this field -- I have removed it now here: https://github.com/project-chip/connectedhomeip/pull/15367/commits/82a749ffe52646ba7422a171d29fa8f53d9b7184

The storageOption is indeed set to "external" for the Basic Cluster: https://github.com/ti-tpan/connectedhomeip/blob/ti/ffs/examples/lock-app/lock-common/lock-app.zap#L1292